src/Pos/Delegation/Logic/VAR.hs

```diff
@@ -341,7 +361,11 @@ dlgVerifyBlocks blocks = do
         HashSet StakeholderId ->
         Block ssc ->
         ExceptT Text (MapCede (StateT DlgVerState m)) DlgUndo
-    verifyBlock _ (Left _) = [] <$ (dvCurEpoch .= HS.empty)
+    verifyBlock _ (Left genesisBlk) = do
+        dvCurEpoch .= HS.empty
+        let blkEpoch = genesisBlk ^. epochIndexL
+        noLongerRichmen <- getNoLongerRichmen blkEpoch
+        catMaybes <$> mapM getPsk noLongerRichmen
```

Comment:
I think you also need to remove it from `Cede` context (`modPsk`). Or not. Depending on whether we implement "don't verify over genesis block" policy. If we do, you should.